custom_components/foxess_modbus/entities/xx1_aux_sensors.py

```diff
@@ -82,6 +82,42 @@
         native_unit_of_measurement="kWh",
         scale=0.1,
     ),
+    SensorDescription(
+        key="charge_total",
```

Comment:
@nathanmarlor The sensors for current power are called `battery_charge` and `battery_discharge`. Would it be worth calling these `battery_charge_total` etc to match?